core/notifications/subgraph/schema.graphql

```diff
@@ -1,15 +1,29 @@
+type InAppNotification {
+	id: ID!
+	title: String!
+	body: String!
+	deepLink: String
+	createdAt: Timestamp!
+	readAt: Timestamp
+}
+
+
 type Mutation {
 	userDisableNotificationChannel(input: UserDisableNotificationChannelInput!): UserUpdateNotificationSettingsPayload!
 	userEnableNotificationChannel(input: UserEnableNotificationChannelInput!): UserUpdateNotificationSettingsPayload!
 	userDisableNotificationCategory(input: UserDisableNotificationCategoryInput!): UserUpdateNotificationSettingsPayload!
 	userEnableNotificationCategory(input: UserEnableNotificationCategoryInput!): UserUpdateNotificationSettingsPayload!
+	userMarkInAppNotificationAsRead(input: UserMarkInAppNotificationAsReadInput!): UserMarkInAppNotificationAsReadPayload!
```

Comment:
Is the subgraph not exposed or just not used?

Reply:
Currently not exposed at all - we are proxying all calls from main schema.

Reply:
Should we just delete it all for now since we aren't currently using it? They also seem to be named incorrectly.

Reply:
We want to use it (as it would save us a bunch of code in core).

You suggested starting with the proxying solution to have more time to asses the correct schema as it is a breaking change which needs coordinating with mobile app release.

So we can delete it to accelerate bringing out this feature but it still needs revisiting.

Reply:
Removed it for now, but agree we will need to bring it back.